File: tests/Generator/resource/java/complex/AnyPropertyType.java
package org.typeapi.model;

import com.fasterxml.jackson.annotation.*;

@JsonClassDescription("Represents an any value which allows any kind of value")
public class AnyPropertyType extends PropertyType {
    private String type = "any";

    @JsonSetter("type")
    public void setType(String type) {
        this.type = type;
    }

    @JsonGetter("type")
    public String getType() {
        return this.type;
    }
}

File: tests/Generator/resource/csharp/complex/AnyPropertyType.cs
using System.Text.Json.Serialization;

namespace TypeAPI.Model;

/// <summary>
/// Represents an any value which allows any kind of value
/// </summary>
public class AnyPropertyType : PropertyType
{
    [JsonPropertyName("type")]
    public new string? Type { get; set; } = "any";

}

